modules/connecttest.lua

```diff
@@ -171,7 +171,12 @@ function Test:runSDL()
     SDL.autoStarted = false
     return
   end
-  local result, errmsg = SDL:StartSDL(config.pathToSDL, config.SDL, config.ExitOnCrash)
+  if config.remoteConnection.enabled then
```

Comment:
@AStarodub  I would propose next code instead to avoid duplication and to fix logic:
```
local result, errmsg = SDL:StartSDL(config.pathToSDL, config.SDL, config.ExitOnCrash)
if result then
  if config.remoteConnection.enabled then
    self.applinkConnection:sendSignal(remote_application_utils.SDLMsgType.SDL_MSG_SDL_START)
  end
else
  quit(exit_codes.aborted)
end
```

Reply:
good suggestion, done